krun/scheduler.py

```diff
@@ -225,6 +217,19 @@ def build_schedule(self, config, current_result_json):
         if self.resume and current_result_json is not None:
             self._remove_previous_execs_from_schedule(current_result_json)
 
+            # Load back (and sanity check) ETA estimates
+            self.eta_estimates = current_result_json["eta_estimates"]
+            for key, exec_data in current_result_json["data"].iteritems():
+                got_len = len(current_result_json["eta_estimates"][key])
+                expect_len = len(exec_data)
+                if expect_len != got_len:
+                    msg = "ETA estimates didn't tally with results: "
+                    msg += "key=%s, expect_len=%d, got_len=%d" % \
+                        (key, expect_len, got_len)
+                    util.log_and_mail(self.mailer, error,
+                                      "Fatal Krun Error",
```

Comment:
I notice you have used `util.log_and_mail()` to report a fatal error and I have used `fatal()` just below. Should these be consistent?